Button/Button.js

```diff
@@ -279,10 +281,10 @@ const ButtonDecorator = compose(
  * Usage:
  * ```
  * <Button
- * 	backgroundOpacity="translucent"
- * 	color="blue"
+ *   backgroundOpacity="translucent"
+ *   color="blue"
```

Comment:
```suggestion
 *   backgroundOpacity="transparent"
```

Reply:
Will be fixed.

Reply:
Good catch. Agate/Button doesn't support `color` prop.

Reply:
Honestly, component reviewers need to be more careful when they review the code.

Reply:
`color="blue"` need to be removed.

Reply:
Will be fixed